war/src/main/less/pluginSetupWizard.less

```diff
@@ -544,12 +552,24 @@
 
         .dependent {
             color: #aaa;
-            padding-left: .4em;
+            padding-left: 10px;
 
             &:before {
                 content: ' ** ';
             }
         }
+        
+        .dependency-legend {
+            color: #aaa;
```

Comment:
@kzantow If the price to get more contrast here is more contrast for dependencies in the list, seems like an improvement all around.